### PR TITLE
Disable inbound iptables capture on standard ports for VMs

### PR DESCRIPTION
**Pull request by howardjohn**
Fixes https://github.com/istio/istio/issues/29412

cc @stevenctl @douglas-reid @liminw

### CODE REVIEW

**Review by liminw — APPROVED** [review]
Thanks for the fix!

**Review by douglas-reid — APPROVED** [review]

File: istioctl/cmd/testdata/vmconfig/simple/cluster.env.golden [hunk 1]
```diff
@@ -2,6 +2,7 @@ CANONICAL_REVISION='latest'
 CANONICAL_SERVICE='foo'
 CLUSTER_MESH_CONFIG_VALUE='foo'
 ISTIO_INBOUND_PORTS='*'
+ISTIO_LOCAL_EXCLUDE_PORTS='15090,15021'
```
Comment (liminw) [inline istioctl/cmd/testdata/vmconfig/simple/cluster.env.golden:5]: How about 15020? Should it also be excluded?

Comment (liminw) [reply istioctl/cmd/testdata/vmconfig/simple/cluster.env.golden:5]: I applied "ISTIO_LOCAL_EXCLUDE_PORTS='15090,15021" in cluster.env for vm. I saw a lot of "[2021-03-31T21:42:51.778Z] "- - -" 0 NR filter_chain_not_found - "-" 0 0 0 - "-" "-" "-" "-" "-" - - 21.0.100.69:15020 21.0.100.66:56926 - -" in the log.

Comment (howardjohn) [reply istioctl/cmd/testdata/vmconfig/simple/cluster.env.golden:5]: You should have set `"ISTIO_LOCAL_EXCLUDE_PORTS='15090,15021"`. That is what `istioctl x workload` will generate

**Review by stevenctl — APPROVED** [review]

**Review by shamsher31 — APPROVED** [review]

**Pull request merged**